conans/test/functional/toolchains/cmake/test_cmake_toolchain.py

```diff
@@ -240,7 +240,7 @@ class Conan(ConanFile):
 
     client.run("create dep")
     client.run("install .")
-    client.run_command("cmake . -DCMAKE_TOOLCHAIN_FILE=./conan_toolchain.cmake "
+    client.run_command("cmake . -DCMAKE_TOOLCHAIN_FILE=./conan_toolchain.cmake -DCMAKE_BUILD_TYPE=Release"
```

Comment:
Maybe only add this for Linux and Mac, but not Windows? 
My concern is that we could be forcing to define it, even for multiconfig if there is some bug there, because in our testing we keep passing CMAKE_BUILD_TYPE also for multiconfig